fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1048,6 +1048,16 @@ ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self
 ACTOR Future<Void> assignMutationsToStorageServers(CommitBatchContext* self) {
 	state ProxyCommitData* const pProxyCommitData = self->pProxyCommitData;
 	state std::vector<CommitTransactionRequest>& trs = self->trs;
+	state std::set<ptxn::StorageTeamID> privateTeamsForServersInSeedTeam;
+	if (SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS && self->commitVersion == 1) {
```

Comment:
I'd create a function like `self->isNewCluster()` that returns `self->commitVersion == 1` to be more understandable.